pkg/cmd/server/logs.go

```diff
@@ -28,27 +36,130 @@ var logsCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		query := ""
-		if followFlag {
-			query = "follow=true"
+		//implement the logging of logs here
+		renderLogs(activeProfile)
+
+	},
+}
+
+func checkFileExists(filePath string) bool {
+	_, err := os.Stat(filePath)
+	if err == nil {
+		return true
+	}
+	if os.IsNotExist(err) {
+		return false
+	}
+	return true // This handles other errors, assuming the file exists
+}
+
+func renderLogs(activeProfile config.Profile) {
```

Comment:
No need for this function. It's not reused so please move it to the Run command above.